client/my-sites/hosting/phpmyadmin-card/index.js

```diff
@@ -0,0 +1,78 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import CardHeading from 'components/card-heading';
+import MaterialIcon from 'components/material-icon';
+import Button from 'components/button';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getHttpData, requestHttpData } from 'state/data-layer/http-data';
+import { http } from 'state/data-layer/wpcom-http/actions';
+
+const requestId = siteId => `pma-link-request-${ siteId }`;
+
+export const requestPmaLink = siteId =>
+	requestHttpData(
+		requestId( siteId ),
+		http(
+			{
+				method: 'GET',
+				path: `/sites/${ siteId }/hosting/pma`,
+				apiNamespace: 'wpcom/v2',
+			},
+			{}
+		),
+		{
+			fromApi: () => ( { pmaUrl } ) => {
+				return [ [ requestId( siteId ), { pmaUrl } ] ];
```

Comment:
if all we want is the link then it's probably fine to return `pmaUrl` instead of `{ pmaUrl }`; just an idea

Reply:
thanks! i wasn't sure if the object was necessary, so that's helpful